tap_stripe/schemas/disputes.json

```diff
@@ -0,0 +1,205 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "properties":{
+    "id": {
+      "type": [
+        "string"
+      ]
+    },
+    "object": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "amount": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "balance_transaction": {
```

Comment:
This is not a real field anymore. It' still there in the test data, so I've left it here for now so the test passes. However, I reached out to Stripe on their IRC channel and they said they created a task to fix it.

I imagine it's best to not accept the PR with this property included in the schema since it doesn't reflect real life. What do you guys think?

Reply:
see: https://stripe.com/docs/upgrades#2014-08-20

Reply:
lets filter this out of the responses with a code comment noting that we're sure it's deprecated and being returned erroneously

Reply:
so actually, I realized the error I was getting was on `balance_transactions` because I had them of type `string`. Once I switched that to type object it works fine. I can omit `balance_transaction` with no negative side effects since it'll just be ignored.